src/btree/bt_cursor.c

```diff
@@ -558,15 +588,18 @@ __wt_btcur_search_near(WT_CURSOR_BTREE *cbt, int *exactp)
 			exact = -1;
 	}
 
+err:	if (exactp != NULL && (ret == 0 || ret == WT_NOTFOUND))
+		*exactp = exact;
```

Comment:
I think it's a mistake -- a change went in about 4 years ago in 0572f6c that looks incorrect to me, there's no reason we should be testing `ret == WT_NOTFOUND`.

The test suite passes when that test is removed.